internal/testplanet/storagenode.go

```diff
@@ -71,9 +72,11 @@ func (planet *Planet) newStorageNodes(count int, whitelistedSatellites storj.Nod
 					},
 				},
 			},
-			Operator: storagenode.OperatorConfig{
-				Email:  prefix + "@mail.test",
-				Wallet: "0x" + strings.Repeat("00", 20),
+			Kademlia: kademlia.Config{
+				Operator: kademlia.OperatorConfig{
+					Email:  prefix + "@mail.test",
+					Wallet: "0x" + strings.Repeat("00", 20),
+				},
```

Comment:
are we only adding this back to the storagenode? but not the satellite? for ex, originally the kad.operator.config was in both https://github.com/storj/storj/pull/2966/files#diff-7a0c41e7d3651613555d7d24a6d785dbL89

Reply:
since we run the satellites, I think this should be okay

Reply:
As far as I know, it's okay to completely remove kademlia from the satellite configs. I think the issue was that by moving the operator configs, the SNOs would have to reconfigure